packages/host-arch-arm32/host-arch-arm32.1/opam

```diff
@@ -14,3 +14,6 @@ license: "CC0-1.0+"
 homepage: "https://opam.ocaml.org"
 bug-reports: "https://github.com/ocaml/opam-repository/issues"
 conflict-class: "ocaml-host-arch"
+# Temporary, while these packages are not being installed by the compilers on
+# non-Windows builds.
+available: os = "win32" | arch = "arm32"
```

Comment:
Currently in opam-repository, when you create a 4.13+ switch, you get the appropriate host-arch- package - which is either a specially-determined one for Windows (which does _not_ depend on the `arch` variable) or, for Unix, you get the package corresponding to the `arch` global variable. The `available` constraint here is designed to leave the selection of the `host-arch-` package to the compiler package when `os = "win32"` (which is important - for x86_32 and x86_64 that doesn't necessarily correspond with the `arch` variable) while stopping a user from managing to manually install, say, `host-arch-s390x.1` on macOS, just because that's confusing!

In particular, it means that _depending_ on, say, `host-arch-arm64` can still be used to indicate that a package _requires_ arm64 support on Unix. What you can't do with this PR anymore is _conflict_ `host-arch-arm64` on Unix.

Reply:
Oh ok, that makes more sense. Thanks.